File: Unidad 5 Java/Programa1U5.java
package com.mycompany.programa1U5;

public class Programa1U5 {

    public static void main(String[] args) {
        System.out.println("Suma ="+suma(7,8));
        System.out.println("Resta ="+resta(7,8));
        System.out.println("Multiplicacion ="+multiplicacion(7,8));
        System.out.println("Divicion ="+divicion(7,8));
    }
    public static int suma(int a, int b){
        int r=a+b;
        return r;
    }
    public static int resta(int a, int b){
        int r=a-b;
        return r;
}
    public static int multiplicacion(int a, int b){
        int r=a*b;
        return r;
}
    public static int division(int a, int b){
        int r=a/b;
        return r;
}
}

File: Unidad 5 C#/Programa1U5.cs
using System;

namespace Programa1U4
{
    internal class Programa1U4
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Suma =" + Suma(7, 8));
            Console.WriteLine("Resta =" + Resta(7, 8));
            Console.WriteLine("Multiplicacion =" + Multiplicacion(7, 8));
            Console.WriteLine("Division =" + Division(7, 8));
        }

        public static int Suma(int a, int b)
        {
            int r = a + b;
            return r;
        }

        public static int Resta(int a, int b)
        {
            int r = a - b;
            return r;
        }

        public static int Multiplicacion(int a, int b)
        {
            int r = a * b;
            return r;
        }

        public static int Division(int a, int b)
        {
            int r = a / b;
            return r;
        }
    }
}
